lib/arbre/html/tag.rb

```diff
@@ -116,24 +117,20 @@ def indent(open_tag, child_content, close_tag)
         spaces = ' ' * indent_level * INDENT_SIZE
 
         html = ""
-
         if no_child? || child_is_text?
           if self_closing_tag?
-            html << spaces << open_tag.sub( />$/, '/>' )
+            html += open_tag.sub(/>$/, '/>')
           else
             # one line
-            html << spaces << open_tag << child_content << close_tag
+            html += open_tag + child_content + close_tag
           end
         else
           # multiple lines
-          html << spaces << open_tag << "\n"
-          html << child_content # the child takes care of its own spaces
-          html << spaces << close_tag
+          # the child takes care of its own spaces
+          html += open_tag + "\n" + child_content + spaces + close_tag
         end
 
-        html << "\n"
-
-        html
+        spaces + html + "\n"
```

Comment:
I'd rather limit the changes here to just change `<<` to either `+=` or `+` as necessary but otherwise keep the same logic of building everything in order in the local variable `html` and return the final result.

Reply:
Good feedback.  Thank you!